packages/react-dom/src/events/__tests__/SyntheticMouseEvent-test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+let React;
+let ReactDOM;
+
+describe('SyntheticMouseEvent', () => {
+  let container;
+
+  beforeEach(() => {
+    React = require('react');
+    ReactDOM = require('react-dom');
+
+    // The container has to be attached for events to fire.
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    document.body.removeChild(container);
+    container = null;
+  });
+
+  it('should only use values from movementX/Y when event type is mousemove', () => {
+    const events = [];
+    const onMouseMove = event => {
+      events.push(event.movementX);
+    };
+
+    const onMouseDown = event => {
+      events.push(event.movementX);
+    };
+
+    const node = ReactDOM.render(
+      <div onMouseMove={onMouseMove} onMouseDown={onMouseDown} />,
+      container,
+    );
+
+    let event = new MouseEvent('mousemove', {
+      relatedTarget: null,
+      bubbles: true,
+      screenX: 0,
```

Comment:
Let's set these to non-zero values? Then the test won't be fooled e.g. by always returning `screenX`.